examples/combined_ODNP.py

```diff
@@ -309,13 +309,26 @@
             os.remove("temp_ODNP.h5")
             DNP_data.hdf5_write(filename, directory=target_directory)
             final_log.append(
-                "if I got this far, that probably worked -- be sure to move/rename temp_ODNP.h5 to the correct name!!"
+                "if I got this far, that probably worked -- be sure to move/rename temp_ODNP.h5 to the correct name!!")
     logger.info("\n*** FILE SAVED IN TARGET DIRECTORY ***\n")
-    logger.debug(strm("Name of saved data", echo_data.name()))
+    logger.debug(strm("Name of saved data", DNP_data.name()))
     # }}}
     # {{{run IR
+    last_dB_setting = 10
     for j, this_dB in enumerate(T1_powers_dB):
-        p.set_power(this_dB)
+        # {{{ make small steps in power if needed
+        if j == 0:
+            p.set_power(this_dB)
```

Comment:
The part in the if block shouldn't be needed. The else should work given that you set last power to 10 above

Reply:
I think you were referring to the `if this_dB == 0`? so I edited and tested with ipython and this runs well

Reply:
I'm talking about `if j==0`